triangle_facts/triangle_facts.rb

```diff
@@ -1,57 +1,68 @@
 # Killer facts about triangles AWW YEAH
 class Triangle
-	attr_accessor :side1,:side2,:side3
 
-	def initialize(side1,side2,side3)
-		@side1,@side2,@side3 = side1,side2,side3
-	end
+	attr_accessor :side1, :side2, :side3
 
-	def equilateral()
-		return side1 == side2 && side2 == side3
+	def initialize(side1, side2, side3)
+		@side1 = side1
+		@side2 = side2
+	 	@side3 = side3
 	end
 
-	def isosceles()
-		return [side1,side2,side3].uniq.length == 2
-	end
-
-	def scalene()
-		unless ! (equilateral || isosceles)
-	    false
-	  else
-	    true
-	  end
-	end
 
 	def recite_facts
-		puts 'This triangle is equilateral!' if equilateral
-		puts 'This triangle is isosceles! Also, that word is hard to type.' if isosceles 
-		puts 'This triangle is scalene and mathematically boring.' if scalene 
-
-		angles = self.calculate_angles(side1,side2,side3)
-		puts 'The angles of this triangle are ' + angles.join(',')
-
-		puts 'This triangle is also a right triangle!' if angles.include? 90
-		puts ''
+		angles = self.calculate_angles(side1, side2, side3)
+		puts "The angles of this triangle are #{angles.join(', ')}"
+		case
+		when equilateral
+			puts "This triangle is equilateral!"
+		when isosceles
+			puts "This triangle is isosceles! Also, that word is hard to type."
+		when scalene
+			puts "This triangle is scalene and mathematically boring."
+		when angles.include?(90)
```

Comment:
Nitpick: code reads nicer when things are on the same level of abstraction. `equilateral`, `isosceles`, `scalene` are all siblings and then we have a combo breaker on `angles.include?(90)`